docs/Map-Hooks.md

```diff
@@ -1,6 +1,7 @@
 HuskTowns has optional support for integrating with popular server web map plugins. The following map plugins are supported:
 * 🗺️ [Dynmap](https://github.com/webbukkit/dynmap) 
 * 💙 [BlueMap](https://www.spigotmc.org/resources/bluemap.83557/)
+* 💙 [Pl3xMap](https://modrinth.com/plugin/pl3xmap)
```

Comment:
It'd probably be easier to just remove these emojis at this point, heh. 😅

Reply:
I updated the emoji in [888126a](https://github.com/WiIIiam278/HuskTowns/pull/303/commits/888126af9e544c1522fa87b179bdb09de672b264) nonetheless